homeassistant/components/nest/__init__.py

```diff
@@ -173,18 +180,29 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
             )
         )
         return False
+    except ConfigurationException as err:
+        if DATA_SUBSCRIBER not in hass.data[DOMAIN].get("A", {}):
```

Comment:
This looks weird. Is it a typo?

Reply:
We won't try to set up again here. Do we still need the guard?

Reply:
Yes, you are right, this should be removed.